keyboards/cozykeys/speedo/config.h

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
I think this file can actually be deleted too? I tried it locally and things still build

Reply:
Yes, since the v2 and v3 folders both have a config.h which includes config_common.h. 👍